src/components/container/ProviderSearch/ProviderSearch.tsx

```diff
@@ -51,12 +55,20 @@ export default function (props: ProviderSearchProps) {
 	}
 
 	function performSearch(search: string) {
+		setSearching(true);
 		let requestID = ++currentRequestID;
 
-		MyDataHelps.getExternalAccountProviders(search, null, 0, 0).then(function (searchResults) {
+		MyDataHelps.getExternalAccountProviders(search, null, props.pageSize, currentPage).then(function (searchResultsResponse) {
 			if (requestID == currentRequestID) {
-				// @ts-ignore
-				setSearchResults(searchResults.filter(a => props.providerCategories?.indexOf(a.category) != -1));
+				if (props.pageSize === 0) {
```

Comment:
I would not allow pageSize of 0 anymore at all - it's just a poor UX since the list is too long to load in it's entirety.